aws/alb.go

```diff
@@ -331,11 +335,23 @@ func findManagedLoadBalancers(svc elbv2iface.ELBV2API, clusterID string) ([]*Loa
 	return loadBalancers, err
 }
 
+func getListeners(alb elbv2iface.ELBV2API, loadBalancerARN string) ([]*elbv2.Listener, error) {
+	// TODO: paged results
```

Comment:
Any particular reason not to go with http://docs.aws.amazon.com/sdk-for-go/api/service/acm/#ACM.ListCertificatesPages ?